solidity/contracts/BondedECDSAKeep.sol

```diff
@@ -593,6 +436,164 @@ contract BondedECDSAKeep is IBondedECDSAKeep {
         require(success, "Transfer failed");
     }
 
+    /// @notice Initialization function.
+    /// @dev We use clone factory to create new keep. That is why this contract
+    /// doesn't have a constructor. We provide keep parameters for each instance
+    /// function after cloning instances from the master contract.
+    /// @param _owner Address of the keep owner.
+    /// @param _members Addresses of the keep members.
+    /// @param _honestThreshold Minimum number of honest keep members.
+    /// @param _tokenStaking Address of the TokenStaking contract.
+    /// @param _keepBonding Address of the KeepBonding contract.
+    /// @param _keepFactory Address of the BondedECDSAKeepFactory that created
+    /// this keep.
+    function initialize(
```

Comment:
Moving this function below `external` ones hurts me because this is like a constructor that we use in the factory pattern.